docs/setup/storage/s3.md

```diff
@@ -40,7 +29,7 @@ or the [S3 gateway](../../integrations/spark.md#access-lakefs-using-the-s3a-gate
             "s3:ListMultipartUploadParts"
          ],
          "Effect": "Allow",
-         "Resource": ["arn:aws:s3:::<STORAGE_NAMESPACE>/*"],
+         "Resource": ["arn:aws:s3:::<BUCKET_NAME>/*"],
```

Comment:
This should be `STORAGE_NAMESPACE`. This is to limit the actions to work on the exact lakeFS repository location

Reply:
To use @kesarwam's "crawl - walk - run" analogy, this page is IMO still in the "crawl" part. We want to get the user going as fast as possible. At this point, we're not sure they have already grasped the "storage namespace" term.

Reply:
Ok.
To minimize issues and questions from users I would add a comment such as:

> To minimize lakeFS access to your bucket, replace `<BUCKET_NAME>` with your lakeFS repository complete [storage namespace](../../understand/model.md#concepts-unique-to-lakefs)